To: vim_dev@googlegroups.com
Subject: Patch 7.3.090
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.090
Problem:    Wrong help text for Cscope.
Solution:   Adjust the help text for "t". (Dominique Pelle)
Files:	    src/if_cscope.c


*** ../vim-7.3.089/src/if_cscope.c	2010-12-17 20:23:56.000000000 +0100
--- src/if_cscope.c	2010-12-30 11:35:37.000000000 +0100
***************
*** 1346,1352 ****
  		       "       g: Find this definition\n"
  		       "       i: Find files #including this file\n"
  		       "       s: Find this C symbol\n"
! 		       "       t: Find assignments to\n"));
  
  	cmdp++;
      }
--- 1346,1352 ----
  		       "       g: Find this definition\n"
  		       "       i: Find files #including this file\n"
  		       "       s: Find this C symbol\n"
! 		       "       t: Find this text string\n"));
  
  	cmdp++;
      }
***************
*** 1657,1663 ****
  /*
   * PRIVATE: cs_make_vim_style_matches
   *
!  * convert the cscope output into into a ctags style entry (as might be found
   * in a ctags tags file).  there's one catch though: cscope doesn't tell you
   * the type of the tag you are looking for.  for example, in Darren Hiebert's
   * ctags (the one that comes with vim), #define's use a line number to find the
--- 1657,1663 ----
  /*
   * PRIVATE: cs_make_vim_style_matches
   *
!  * convert the cscope output into a ctags style entry (as might be found
   * in a ctags tags file).  there's one catch though: cscope doesn't tell you
   * the type of the tag you are looking for.  for example, in Darren Hiebert's
   * ctags (the one that comes with vim), #define's use a line number to find the
*** ../vim-7.3.089/src/version.c	2010-12-24 14:00:09.000000000 +0100
--- src/version.c	2010-12-30 11:36:33.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     90,
  /**/

-- 
ERROR 047: Keyboard not found.  Press RETURN to continue.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
